addons/project/models/project_task.py

```diff
@@ -1087,6 +1088,38 @@ def update_date_end(self, stage_id):
             return {'date_end': fields.Datetime.now()}
         return {'date_end': False}
 
+    def search_on_comodel(self, domain, field, comodel, order=None, additional_domain=None):
```

Comment:
Should be private since we will not call client side (JS) 🙂 
```suggestion
    def _search_on_comodel(self, domain, field, comodel, order=None, additional_domain=None):
```

cc @auon-odoo 🙂